crates/core/src/estimation.rs

```diff
@@ -0,0 +1,188 @@
+use crate::db::relational_db::Tx;
+use spacetimedb_primitives::{ColList, TableId};
+use spacetimedb_vm::expr::{Query, QueryExpr, SourceExpr};
+
```

Comment:
The way this is written is to me not ` estimated number of rows` but `MAXIMUM estimated number of rows`.

Reply:
This is true, although the plan is that this will become more accurate over time as we add more stats.